src/content/docs/new-relic-control/agent-control/agent-types.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Supported Agent Types"
+metaDescription: "Types of Agents that can be managed by Agent Control"
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+## Overview
+
+Agent Control provides a single, unified platform for managing a wide variety of New Relic and OpenTelemetry agents across your fleets. By centralizing management, you can deploy, configure, and update your instrumentation with ease.
+
+While Agent Control is designed to support both containerized and on-premise environments, support for specific agent types varies. The following table provides a comprehensive overview of which agents can be managed by Agent Control and their support status across environments.
+
+### Current Support
+
+| Agent Type | Purpose | Kubernetes Support | On-host Linux Support |
+| :--- | :--- | :---: | :---: |
+| **New Relic Infrastructure Agent** | Collects system-level metrics from your hosts and containers. | ✅ Yes | ✅ Yes (Public Preview) |
+| **New Relic OpenTelemetry Collector (NRDOT)** | Gathers metrics, logs, and traces from various sources and exports them to New Relic. | ✅ Yes | ✅ Yes (Public Preview) |
+| **Fluent Bit** | A lightweight log processor and forwarder. | ✅ Yes | 🚫 No |
+| **New Relic Prometheus Agent** | Scrapes Prometheus-format metrics from your services and forwards them to New Relic. | ✅ Yes | 🚫 No |
+| **New Relic eBPF Agent** | Uses eBPF technology for high-resolution, deep visibility into system activity. | ✅ Yes | 🚫 No |
+| **APM Agents (.NET, Java, Node, Python, Ruby)** | Instruments your applications to monitor their performance and track transactions. | 🚫 No | 🚫 No |
+
+## Agent details
+
+### New Relic APM Agents
+
+APM agents instrument your applications to monitor their performance, track transactions, and provide code-level visibility. Agent Control allows you to manage these agents across your containers and hosts, ensuring consistent monitoring across your application portfolio.
+
+<Callout title="warning">
+  APM agents are currently not supported by Agent Control. We are working on it!
+</Callout>
+
+### Infrastructure, Logging, and Metrics Agents
+These agents are designed for collecting telemetry from your hosts, containers, and services, providing deep visibility into your entire infrastructure.
+
+#### New Relic Infrastructure Agent
+
+- Type ID: newrelic/com.newrelic.infrastructure:0.1.0
```

Comment:
I wonder if we could avoid mentioning specifics such us the type ID which could get outdated 🤔